ambry-store/src/test/java/com.github.ambry.store/IndexTest.java

```diff
@@ -1317,7 +1331,8 @@ private void verifyValue(MockId id, IndexValue valueFromFind) {
       assertNotNull("Value should be successfully fetched", valueFromFind);
       IndexValue expectedValue = state.getExpectedValue(id, !state.deletedKeys.contains(id));
       assertEquals("Offset in value from index not as expected", expectedValue.getOffset(), valueFromFind.getOffset());
-      assertEquals("Bytes from value from index not as expected", expectedValue.getBytes(), valueFromFind.getBytes());
+      assertEquals("Bytes from value from index not as expected for " + id + " ex " + expectedValue + ", actual "
```

Comment:
yes. for informational purposes. If the assertion fails, byte[] ref will be printed which may not be useful. So, added this.